anime_downloader/config.py

```diff
@@ -80,12 +81,6 @@
             "servers": ["vidstream","mp4upload","trollvid"],
             "version": "dubbed",
         },
-        'animedaisuki': {
-            "servers": ["official"]
-        },
-        'vidstream': {
```

Comment:
your git history is broken

Reply:
wait, what does that have to do with the 4 lines you quoted?